examples/s3-import-mysql/main.tf

```diff
@@ -182,13 +174,13 @@ module "db" {
 
   # S3 import https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Procedural.Importing.html
   s3_import = {
-    source_engine_version = "8.0.27"
-    bucket_name           = aws_s3_bucket.import.id
+    source_engine_version = "8.0.28"
+    bucket_name           = module.import_s3_bucket.s3_bucket_id
     ingestion_role        = aws_iam_role.s3_import.arn
   }
 
   multi_az               = true
-  subnet_ids             = module.vpc.database_subnets
+  db_subnet_group_name   = module.vpc.database_subnet_group_name
```

Comment:
Fixes this error: 
<img width="1273" alt="Screen Shot 2022-07-24 at 9 58 58 PM" src="https://user-images.githubusercontent.com/69476188/180678695-d932babd-964e-4aec-9424-9c89ae49ac5a.png">